app/messages.py

```diff
@@ -9,7 +9,9 @@
 FIELD_AVAILABLE_TO_MENTOR_IS_INVALID = {
     "message": "Field available_to_mentor" " is not valid."
 }
-
+USERNAME_FIELD_HAS_INVALID_LENGTH={
```

Comment:
```suggestion
NAME_FIELD_HAS_INVALID_LENGTH={
```
This is to avoid confusion with USERNAME :)

Reply:
@isabelcosta -  thanks for the suggestions, will incorporate the same and push the changes